CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/gui/VisualGraphOpener.java

```diff
@@ -176,12 +178,45 @@ protected Void doInBackground() throws Exception {
             if (graph == null) {
                 try {
                     final long t0 = System.currentTimeMillis();
+                    LOGGER.log(Level.INFO, String.format("MMDEBUG Attempting to open %s", graphFile.toString()));
                     graph = new GraphJsonReader().readGraphZip(graphFile, new HandleIoProgress(String.format("Reading %s...", graphFile.getName())));
                     time = System.currentTimeMillis() - t0;
-                } catch (GraphParseException | IOException | RuntimeException ex) {
+                } catch (GraphParseException | IOException | RuntimeException ex) {  
```

Comment:
Noting you didn't add this but if could make this final as well that would be great

Reply:
as above